components/Seed/Seed.tsx

```diff
@@ -58,9 +62,33 @@ const Seed: React.FunctionComponent<SeedProps> = ({ onClickOK, onClickCancel, ac
     );
     setTimes(action === 'change' || action === 'backup' || action === 'server' ? 1 : 0);
     setSeedPhrase(walletSeed?.seed || '');
-    setBirthdayNumber(walletSeed?.birthday || 0);
+    setBirthdayNumber(walletSeed?.birthday?.toString() || '');
```

Comment:
What happens when `Number('')` receives an empty string?

Reply:
The value in TextInput is always a string and if this value are empty, the field shows the placeholder...